tests/Features/Skip.php

```diff
@@ -17,11 +17,15 @@
     ->assertTrue(false);
 
 it('skips with truthy closure condition')
-    ->skip(function () { return '1'; })
+    ->skip(function () {
+        return '1';
+    })
     ->assertTrue(false);
 
 it('do not skips with falsy closure condition')
-    ->skip(function () { return false; })
+    ->skip(function () {
+        return false;
+    })
```

Comment:
Looks like this was accidentally reformatted.
```suggestion
    ->skip(function () { return false; })
```